scripts/document_types/trait.py

```diff
@@ -15,9 +15,17 @@ def get_trait_data(connection, limit=0):
     '''
 
     efo_sql = """
-        SELECT DISTINCT(ET.ID), ET.TRAIT, ET.URI, 
-            'trait' as resourcename, ET.SHORT_FORM
-        FROM EFO_TRAIT ET
+        SELECT DISTINCT (ET.ID), ET.TRAIT, ET.URI, 
+            'trait' as resourcename, ET.SHORT_FORM 
+        FROM EFO_TRAIT ET, STUDY_EFO_TRAIT SETR, STUDY S, HOUSEKEEPING H 
+        WHERE ET.ID=SETR.EFO_TRAIT_ID and SETR.STUDY_ID=S.ID 
+            and S.HOUSEKEEPING_ID=H.ID and H.IS_PUBLISHED=1 
```

Comment:
I'm not sure if we have to specify `H.IS_PUBLISHED=1` given the script is called on SPOTREL

Reply:
Good point. The `H.IS_PUBLISHED=1` is included since testing is done on a local database, which has both published and unpublished studies. That condition could be removed, however including it doesn't add significantly to the query timing (240ms vs. 180ms) or overall length of time to run the script so I'd err on the side of leaving the condition.

Reply:
Yes, totally make sense.

Reply:
Great, are any other changes requested before merging?